uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -597,6 +599,34 @@ define([
       });
     }
 
+    /**
+     * Parses value depending on the specified type
+     *
+     * @param  {String} value
+     * @param  {String} type "boolean|decimal|integer"
+     * @return {Boolean|String}
+     */
+    function parseLoadedValue (value, type) {
+      var parsers = {
```

Comment:
it's better to define these parsers outside of the function. Right now they are being redefined each time a field needs to be parsed. I don't think that's optimal.

Reply:
Done.